app/src/main/java/com/team41/boromi/auth/LoginFragment.java

```diff
@@ -0,0 +1,89 @@
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.team41.boromi.BookActivity;
+import com.team41.boromi.R;
+
+/**
+ * A simple {@link Fragment} subclass. Use the {@link LoginFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class LoginFragment extends Fragment {
+
+  // TODO: Rename parameter arguments, choose names that match
+  // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+  private static final String ARG_PARAM1 = "param1";
+  private static final String ARG_PARAM2 = "param2";
+
+  // TODO: Rename and change types of parameters
+  private String mParam1;
+  private String mParam2;
+
+  public LoginFragment() {
+    // Required empty public constructor
+  }
+
+  /**
+   * Use this factory method to create a new instance of this fragment using the provided
+   * parameters.
+   *
+   * @param param1 Parameter 1.
+   * @param param2 Parameter 2.
+   * @return A new instance of fragment LoginFragment.
+   */
+  // TODO: Rename and change types and number of parameters
+  public static LoginFragment newInstance(String param1, String param2) {
+    LoginFragment fragment = new LoginFragment();
+    Bundle args = new Bundle();
+    args.putString(ARG_PARAM1, param1);
+    args.putString(ARG_PARAM2, param2);
+    fragment.setArguments(args);
+    return fragment;
+  }
+
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    if (getArguments() != null) {
+      mParam1 = getArguments().getString(ARG_PARAM1);
+      mParam2 = getArguments().getString(ARG_PARAM2);
+    }
+  }
+
+  @Override
+  public View onCreateView(LayoutInflater inflater, ViewGroup container,
+      Bundle savedInstanceState) {
+    // Inflate the layout for this fragment
+    View view = inflater.inflate(R.layout.fragment_login, container, false);
+    Button recoverPasswordButton = (Button) view.findViewById(R.id.recoverPassword);
+    Button loginButton = (Button) view.findViewById(R.id.login);
+    recoverPasswordButton.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View view) {
+        Fragment recoverPasswordFragment = new RecoverPasswordFragment();
+        FragmentManager manager = getFragmentManager();
+        FragmentTransaction ft = manager.beginTransaction();
+        ft.replace(R.id.auth_fragment, recoverPasswordFragment).addToBackStack("recover").commit();
+      }
+    });
+    loginButton.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View view) {
+        Intent intent = new Intent(getActivity(), BookActivity.class);
+        startActivity(intent);
+      }
+    });
```

Comment:
Add some spacing between the click handlers and give a brief comment of what they do

Reply:
auto formatting 😢